README.md

```diff
@@ -51,7 +51,7 @@ https://github.com/Pythagora-io/gpt-pilot/assets/10895136/0495631b-511e-451b-93d
 # 🔌 Requirements
 
 
-- **Python**
+- **Python >= 3.10**
```

Comment:
why python 3.10? I am using 3.9.6 on my current macbook and it works fine

Reply:
I ran into an (minor) issue with typing and figured that Python 3.10 came out in 2021 so it would not be likely to be used anymore.

```
utils/function_calling.py:5: in <module>
    JsonType = str | int | float | bool | None | list["JsonType"] | dict[str, "JsonType"]
E   TypeError: unsupported operand type(s) for |: 'type' and 'type'
```